webpack.config.js

```diff
@@ -65,10 +66,10 @@ module.exports = {
     }),
   ],
   resolve: {
-    extensions: ['', '.js', '.json'],
-    modulesDirectories: [
-      'src',
-      'node_modules',
+    extensions: ['.js', '.json'],
+    modules: [
+      path.resolve(__dirname, 'node_modules'),
+      path.join(__dirname, 'src'),
```

Comment:
`src` should come before `node_modules` in case one of our local modules shadows a third party module. Hopefully it won't happen but if we give precedence to our local modules it would be less confusing.